code/stable/nopcm/SRS/HTML/NoPCM_SRS.html

```diff
@@ -1440,7 +1440,7 @@ <h3>Instance Models</h3>
                     <tr>
                       <th>Equation</th>
                       <td>
-                        \[\frac{\,d{T_{\text{W}}}}{\,dt}=\frac{1}{{τ_{\text{W}}}} \left({T_{\text{C}}}-{T_{\text{W}}}\left(t\right)\right)\]
+                        \[\frac{\,d{T_{\text{W}}}}{\,dt}+-\frac{1}{{τ_{\text{W}}}} \left({T_{\text{C}}}-{T_{\text{W}}}\left(t\right)\right)=0\]
```

Comment:
Hmm, I think we're going to need to fix this printing problem before we pull in this PR. Too high a risk that it doesn't get fixed 'soon'.

Reply:
Responded in the following: 
https://github.com/JacquesCarette/Drasil/issues/2876#issuecomment-966524430